jenkins/databricks/init_cudf_udf.sh

```diff
@@ -20,11 +20,14 @@
 
 CUDF_VER=${CUDF_VER:-21.12}
 
-# Use mamba to install cudf-udf packages to speed up conda resolve time
+# Need to explictly add conda into PATH environment, to activate conda environment.
+export PATH=/databricks/conda/bin:$PATH
+
 base=$(conda info --base)
-conda create -y -n mamba -c conda-forge mamba
-pip uninstall -y pyarrow
-${base}/envs/mamba/bin/mamba remove -y c-ares zstd libprotobuf pandas
-${base}/envs/mamba/bin/mamba install -y pyarrow=1.0.1 -c conda-forge
-${base}/envs/mamba/bin/mamba install -y -c rapidsai -c rapidsai-nightly -c nvidia -c conda-forge -c defaults cudf=$CUDF_VER cudatoolkit=11.0
-conda env remove -n mamba
+# Create and activate 'cudf-udf' conda env for cudf-udf tests
+conda create -y -n cudf-udf && source activate && conda activate cudf-udf
```

Comment:
We change to create a common 'cudf-udf' conda environment for cudf udf tests, because 

1, Not like 7.3 & 8.2, Databricks 9.1 does not have the default `databricks-ml-gpu` conda ENV

2, In 7.3 & 8.2, the script init_cudf_udf.sh  create the empty MAMBA conda environment, but install everything (using MAMBA)into the  default `databricks-ml-gpu` ENV, this looks odd.

3, We create a common and clean `cudf-udf` ENV specified for the cudf udf test.